### PR TITLE
fixes: karmor probe

### PR DESCRIPTION
**Pull request by Ankurk99**
This PR added some fixes related to PR: https://github.com/kubearmor/kubearmor-client/pull/360

### CODE REVIEW

**Review by daemon1024 — CHANGES_REQUESTED** [review]
This is unrelated. the probe daemonset installer and uninstaller are only triggered when KubeArmor is not present. 

But the probe issue is that it's not able to find kubearmor pod for extracting the information itself, even when KubeArmor is installed.

**Comment by Ankurk99** [comment]
> This is unrelated. the probe daemonset installer and uninstaller are only triggered when KubeArmor is not present.

Yes, I am aware about this. Those changes were made in the previous PR and are reversed in this PR.

> But the probe issue is that it's not able to find kubearmor pod for extracting the information itself, even when KubeArmor is installed.

Please check the change at line 489. It addresses that concern.

File: probe/probe.go [hunk 1]
```diff
@@ -486,7 +486,7 @@ func readDataFromKubeArmor(c *k8s.Client, o Options, nodeName string) (KubeArmor
 	cmdArr := []string{"cat", srcPath}
 	req := c.K8sClientset.CoreV1().RESTClient().
 		Get().
-		Namespace("").
+		Namespace(o.Namespace).
```
Comment (daemon1024) [inline probe/probe.go:489]:
```suggestion
		Namespace(pods.Items[0].Metadata.Namespace).
```

Please refer the PodSpec for exact path but this should solve it.

**Review by daemon1024 — CHANGES_REQUESTED** [review]
My bad, regardless karmor probe shouldn't depend on namespace. It should be able to probe in any namespace. Below requested change should help with that.

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**